src/test/java/core/basesyntax/AppTest.java

```diff
@@ -0,0 +1,127 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.fao.CustomFileReader;
+import core.basesyntax.fao.CustomFileReaderImpl;
+import core.basesyntax.fao.CustomFileWriter;
+import core.basesyntax.fao.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.ReportGenerator;
+import core.basesyntax.model.ReportGeneratorImpl;
+import core.basesyntax.model.convertor.DataConvertor;
+import core.basesyntax.model.convertor.DataConvertorImpl;
+import core.basesyntax.model.handler.BalanceOperation;
+import core.basesyntax.model.handler.OperationHandler;
+import core.basesyntax.model.handler.PurchaseOperation;
+import core.basesyntax.model.handler.ReturnOperation;
+import core.basesyntax.model.handler.SupplyOperation;
+import core.basesyntax.model.strategy.OperationStrategy;
+import core.basesyntax.model.strategy.OperationStrategyImpl;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.ShopServiceImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class AppTest {
```

Comment:
Let’s fix your unit testing. Create separate test classes for each service that you want to test